app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss

```diff
@@ -92,6 +92,18 @@
   }
 }
 
+.gem-c-action-link--blue-arrow {
+  &:before {
+    width: 35px;
+    height: 30px;
+    background: image-url("govuk_publishing_components/action-link-arrow--blue.png");
```

Comment:
I think you forgot to add the PNG to `assets/images`.

Reply:
🤦